src/test/java/no/nav/brukernotifikasjon/schemas/builders/StatusoppdateringAvroTest.java

```diff
@@ -0,0 +1,32 @@
+package no.nav.brukernotifikasjon.schemas.builders;
+
+import no.nav.brukernotifikasjon.schemas.Statusoppdatering;
+import no.nav.brukernotifikasjon.schemas.builders.domain.StatusGlobal;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class StatusoppdateringAvroTest {
```

Comment:
Er det en grunn vi ikke sjekker defaultverdi på sikkerhetsnivå her? Ikke krise for min del altså 😊

Reply:
Nei, det hadde jeg glemt. Lagt til nå :)